fiftyone/core/models.py

```
@@ -1216,6 +1223,79 @@ def compute_embeddings(
+        return None
+
+    if errors:
+        return embeddings  # may contain None, must return as list
+
+    if not embeddings:
+        return np.empty((0, 0), dtype=float)
+
+    return np.stack(embeddings)
+
+
+def _load_sample_point_cloud(sample):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_load_sample_point_cloud`

(ANN202)